src/System.Runtime.Numerics/tests/Complex/ComplexTestSupport.cs

```diff
@@ -11,14 +10,10 @@ namespace ComplexTestSupport
     public static class Support
     {
         private static Random s_random;
-        public static int randomSample;
```

Comment:
Any particular reason you removed this? Seems that it was easier to tweak as a field